guarddog/analyzer/metadata/npm/deceptive_author.py

```diff
@@ -0,0 +1,13 @@
+""" Deceptive Author Detector
+
+Detects when an author of is using a disposable email
+"""
+
+from guarddog.analyzer.metadata.deceptive_author import DeceptiveAuthorDetector
+
+from .utils import get_email_addresses
```

Comment:
seems that you forgot to push this file?

Reply:
This file already exists and was left untouched